gpbft/gpbft.go

```diff
@@ -1332,16 +1331,17 @@ func (c *convergeState) Receive(sender ActorID, table PowerTable, value ECChain,
 
 // FindMaxTicketProposal finds the value with the best ticket, weighted by
 // sender power. Returns an invalid (zero-value) ConvergeValue if no converge is found.
-func (c *convergeState) FindMaxTicketProposal(table PowerTable) ConvergeValue {
+func (c *convergeState) FindMaxTicketProposal(table PowerTable, filter func(ConvergeValue) bool) ConvergeValue {
```

Comment:
I still think we should filter when receiving proposals instead of filtering here.

Reply:
I thought about it, but we should not because of how COMMIT works (the commit round is always active).

Reply:
This is CONVERGE. We skip here: https://github.com/filecoin-project/go-f3/blob/71f7e0d13749f6f4368608640a7cd506ecf3ab3b/gpbft/gpbft.go#L358

Reply:
The thing that dictates sway behaviour is the state of previous commit round.

Reply:
Kuba is right...